src/lib/utils/track/schema.ts

```diff
@@ -310,6 +310,11 @@ export enum ActionNames {
   // dispatch this on search input focus and again any time they type
   ARAnalyticsSearchStartedQuery = "Searched",
   ARAnalyticsSearchItemSelected = "Selected result from search screen",
+
+  /**
+   * Collection page events
+   */
+  ViewMore = "viewMore",
```

Comment:
Pretty generic name that I could foresee used on other screens, but I don't think we have a "common" section or anything like that, so I've stuck it here.